controller/lib/core/control.cpp

```diff
@@ -40,9 +40,9 @@ enum class pid_fsm_state {
   expire_dwell = 4,
 };
 
-// TODO: move this into a mathematical library
-// This function is originally from the Arduino library
-long map(long x, long in_min, long in_max, long out_min, long out_max) {
+// Rescales x from range [in_min, in_max] to [out_min, out_max]
```

Comment:
Please document that this is a replacement of Arduino map() and is subject to the same limitations - eg. uses integer math.

Reply:
Done